bin/release_tag.ts

```diff
@@ -127,27 +170,31 @@ const announceRelease = async (tagName: string, commitId: string): Promise<void>
     const message = `Released tag "${tagName}" based on commit ID "${commitId}".`;
     const payload = account.service.conversation.messageBuilder.createText(WIRE_CONVERSATION, message).build();
     await account.service.conversation.send(payload);
+    await sendRandomGif(account, WIRE_CONVERSATION, 'in the oven');
```

Comment:
Let's send this one before sending `Released tag "${tagName}" based on commit ID "${commitId}"`. Otherwise you might miss the important part as it's not the latest message.